apps/web/src/config/constants/endpoints.ts

```diff
@@ -77,4 +77,4 @@ export const QUOTING_API = `${process.env.NEXT_PUBLIC_QUOTING_API}/v0/quote`
 
 export const FARMS_API = 'https://farms-api.pancakeswap.com'
 
-export const MERCURYO_WIDGET_ID = process.env.NEXT_PUBLIC_MERCURYO_WIDGET_ID || '95a003f2-354a-4396-828a-1126d56e4e13'
+export const MERCURYO_WIDGET_ID = process.env.NEXT_PUBLIC_MERCURYO_WIDGET_ID || '76ba4ff5-2686-4ed4-8666-fadb0d9a5888'
```

Comment:
Is this default id for local development purpose?

Reply:
yeah this default id is for everything other than prod. mercuryo widgets are annoying in that you need to make a new one per domain. so i switched this out so the widget loads on the pr deployment url.